src/app/(sidebar)/transaction/sign/components/SignWithWallet.tsx

```diff
@@ -46,33 +48,31 @@ export const SignWithWallet = ({
   const onSignWithWallet = async () => {
     // remove the previously signed tx from the display
     updateSignedTx("");
+    setSignError("");
+    setSignSuccess("");
 
     await kit.openModal({
       onWalletSelected: async (option: ISupportedWallet) => {
         try {
           kit.setWallet(option.id);
-          const publicKey = await kit.getPublicKey();
+          const { address } = await kit.getAddress();
           const networkType = getWalletNetwork(network.id);
 
-          const { result } = await kit.signTx({
-            xdr: sign.importXdr,
+          const { signedTxXdr } = await kit.signTransaction(sign.importXdr, {
             // You could send multiple public keys in case the wallet needs to handle multi signatures
-            publicKeys: [publicKey],
-            network: networkType,
+            address,
+            networkPassphrase: networkType,
```

Comment:
Maybe we should rename `networkType` to `networkPassphrase` to make it clearer what we're using.

Reply:
will do